guidelines/sc/22/focus-appearance.html

```diff
@@ -23,15 +23,16 @@ <h4>Focus Appearance</h4>
         </li>
     </ol>
 
-    <p>Where a user interface component has active sub-components, if a sub-component receives a focus indicator, these requirements may be applied to the sub-component instead.</p>
-
     <p>Exceptions:</p>
     <ul>
         <li>The focus indicator is determined by the user agent and cannot be adjusted by the author, or</li>
         <li>The focus indicator and the indicator's background color are not modified by the author.</li>
     </ul>
-                
-    <p class="note">Note: Examples of sub-components that may receive a focus indicator are menu items in an opened drop-down menu. However, it may also be possible to indicate user interaction for such sub-components by relying strictly on a visual indication of which item is <em>selected</em>. Where selectable sub-components have no differentiated focus indicator, the visual indicator for sub-component selection is measured against <a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">1.4.11 Non-text Contrast</a> requirements, not against this Criterion.</p>
-    <p class="note">Note: Contrast calculations can be based on colors defined within the technology (such as HTML, CSS and SVG). Pixels modified by user agent resolution enhancements and anti-aliasing can be ignored.</p>
+    
+    <p class="note">What is perceived as the user interface component and any sub-components (to determine enclosure or size) depends on its visual presentation. The visual presentation includes, but is not limited to the component's visible content, border, component-specific background, and effects emanating from the visible component such as shadows or glow effects.</p>
+
+    <p class="note">Examples of sub-components that may receive a focus indicator are menu items in an opened drop-down menu.</p>
```

Comment:
Might need to add an example or change to "An example of a sub-component.... is a menu item.."